src/lib/hooks/useOwnedDomains.ts

```diff
@@ -38,10 +38,13 @@ const useOwnedDomains = (
 			const owned = await sdk.getDomainsByOwner(account);
 			// TODO: Optimize this
 			const domainsPaymentTokenData = owned.map(async ({ id }) => {
-				const paymentToken = await sdk.zauction.getPaymentTokenForDomain(id);
+				// const paymentToken = await sdk.zauction.getPaymentTokenForDomain(id);
 				const paymentTokenInfo: PaymentTokenInfo = {
-					...(await sdk.zauction.getPaymentTokenInfo(paymentToken)),
-					id: paymentToken,
+					...(await sdk.zauction.getPaymentTokenInfo(
+						// TODO :: REPLACE WITH paymentToken
```

Comment:
This is required to allow QA to test all claimed motos appearing in profile issue.